items.json

```diff
@@ -14965,6 +14965,29 @@
         "swingDamage": 0,
         "swingDamageType": "Undefined",
         "swingSpeed": 67
+      },
+      { 
+        "class": "TwoHandedPolearm",
+        "itemUsage": "polearm_couch",

```

Comment:
Update as radical as this should be discussed with me prior. Couches are particularly problematic to cRPG, as we cannot modify the mechanics (cooldown , maximum rotational angles ,  collision detection) . I'd probably want to have a few very long couchable lances that you have to drop when you're finished with and that have moderate amount of damage

Reply:
![image](https://user-images.githubusercontent.com/47090987/202597623-6887659d-e0c2-42c5-bda2-53a65cc980d4.png)